### DISCUSSION TITLE
Unclear specifications of SceneViewer in regards to animation size - crashes above unclear limits

### DISCUSSION THREAD

**Discussion by hybridherbst** [google/model-viewer#3658] [Q&A]
### Description

(not model-viewer)

We have a couple of models where file complexity is medium but animation complexity is medium-high (slightly above 2 minutes of sparse animation).

Above a certain unclear limit, SceneViewer just crashes when opening the file. The behaviour reproduces when putting the same files into https://vr.google.com/scene-viewer-preview, which throws:
![image](https://user-images.githubusercontent.com/2693840/75296196-d135f100-582c-11ea-93fd-150a789a7c12.png)

Would be great to either get some info about allowed animation complexity and/or allowed object count (the file is still under 10 MB, we have successfully loaded big 150 MB files fine in both model-viewer and SceneViewer) or have this filed internally as a bug.

#### Live Demo
I sent the files to @cdata via mail since they are covered by NDA.

### Browser Affected

- [X] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [X] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by cdata** [comment]
Thanks for the report @soraryu 

cc @tpsiaki
